src/sources/docker_logs.rs

```diff
@@ -442,12 +442,15 @@ impl DockerLogsSource {
                 value = self.events.next() => {
                     match value {
                         Some(Ok(mut event)) => {
+                            let byte_size = std::mem::size_of_val(&event);
```

Comment:
I don't think this is right. I left a similar comment on another PR, but `std::mem::size_of_val` gives you the size of the type. What we want here is the size of the HTTP response body from the Docker API.